deeplake/enterprise/dataloader.py

```diff
@@ -200,6 +202,13 @@ def _auto_collation(self):
     def _index_sampler(self):
         return self.sampler
 
+    @property
+    def summary(self):
+        if not self._ignore_errors or self._dataloader is None:
```

Comment:
What is the purpose of this property and why hide the summary if _ignore_errors?